src/mumble/AudioInput.h

```diff
@@ -281,4 +300,9 @@ class AudioInput : public QThread {
 	bool isTransmitting() const;
 };
 
+#define ECHO_CANCEL_DISABLED 0
+#define ECHO_CANCEL_DEFAULT 1
+#define ECHO_CANCEL_SPEEX_MIXED 10
+#define ECHO_CANCEL_SPEEX_MULTICHANNEL 11
```

Comment:
```suggestion
enum EchoCancelOptions {
	ECHO_CANCEL_DISABLED = 0,
	ECHO_CANCEL_DEFAULT = 1,
	ECHO_CANCEL_SPEEX_MIXED = 10,
	CHO_CANCEL_SPEEX_MULTICHANNEL = 11,
};
```

I don't think defines are a good choice for implementing this. If you really don't want to use an `enum` (which I thought we agreed upon on IRC - but maybe this was only after this code has been written), use `constexpr unsigned int` instead.
Ref: https://stackoverflow.com/questions/1637332/static-const-vs-define

And where are these particular values coming from?